installer/GNUmakefile

```diff
@@ -22,7 +22,7 @@ SIGNED_list  ?= apl cnl icl jsl tgl tgl-h
 
```

Comment:
Commit msg: "Based on Mengdong's guidance". Proposal:

ADL and ADL-S binaries share FW build configuration with TGL and TGL-H.

Reply:
@mengdonglin

Reply:
Done